src/netaddress.cpp

```diff
@@ -437,6 +437,9 @@ bool CNetAddr::IsValid() const
         return false;
     }
 
+    if (IsCJDNS() && (m_addr[0] != 0xFC))
+        return false;
+
```

Comment:
nit, same line or use braces per doc/developer-notes.md

    If an `if` only has a single-statement `then`-clause, it can appear
    on the same line as the `if`, without braces. In every other case,
    braces are required, and the `then` and `else` clauses must appear
    correctly indented on a new line.

Reply:
I wanted to follow the same style that the rest of the `if` statements had in the method. Should I change it anyway?

Reply:
Yes, absolutely. The developer notes explicitly state you should not try to mimick surrounding style when writing new code.

Reply:
Done